setup.py

```diff
@@ -12,6 +12,9 @@
         "ape-optimism",  # Needed for Optimism integration
         "ape-polygon",  # Needed for Polygon integration
         "ape-polygon-zkevm",  # Needed for Polygon ZkEVM integration
+        # "ape-scroll",  # Needed for Scroll networks integration
+        # Temporary fix due to version restrictions
+        "ape-scroll @ git+https://github.com/Barabazs/ape-scroll.git@chore/version-restriction",
```

Comment:
If you want to get this through the door, we can comment this out and comment out the test-cases with a TODO.

Reply:
Thank you for the review. 

I pinged the maintainer of ape-scroll on [discord](https://discord.com/channels/922917176040640612/933897283618103316/1258080869382361150). I'll wait for a bit and see if they bump ape-scroll or not. Otherwise I'll just do as you suggested.

Reply:
@antazoey new ape-scroll version is published and I've updated this setup.py to point to it.